src/main/java/no/nav/pto/veilarbportefolje/cv/CvRepository.java

```diff
@@ -55,9 +55,10 @@ public String cvEksistere(AktorId aktoerId) {
                 .execute();
     }
 
-    public void slettCVData(AktorId aktoerId) {
-        SqlUtils.delete(jdbcTemplate, TABLE_NAME)
-                .where(WhereClause.equals(AKTOERID, aktoerId.get()))
+    public void resetHarDeltCV(AktorId aktoerId) {
+        SqlUtils.update(jdbcTemplate, TABLE_NAME)
+                .set(HAR_DELT_CV, false)
```

Comment:
Vil dette sette HAR_DELT_CV til verdien N?
Så at det var litt krøll med tanke på at verdien 0 har kommet inn i databasen i q1

Reply:
https://logs.adeo.no/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:10000),time:(from:'2021-08-02T09:01:00.041Z',to:'2021-08-03T02:42:19.152Z'))&_a=(columns:!(message,envclass,level,application,host),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:envclass,negate:!f,params:(query:q),type:phrase),query:(match_phrase:(envclass:q))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:level,negate:!f,params:(query:Error),type:phrase),query:(match_phrase:(level:Error))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:application,negate:!f,params:(query:veilarbindexer),type:phrase),query:(match_phrase:(application:veilarbindexer)))),index:'96e648c0-980a-11e9-830a-e17bbd64b4db',interval:auto,query:(language:lucene,query:''),sort:!())

Reply:
Ahaa, thank you for finding out this!